### DISCUSSION TITLE
report is stuck in jenkins

### DISCUSSION THREAD

**Discussion by jovancacvetkovic** [LironEr/cypress-mochawesome-reporter#105] [Q&A]
Hi,
I have an issue with the report and cypress tests. All the tests pass but the reports are just stuck.
The only log is :
`Start generate report process
Read and merge jsons from "/data1/var/lib/jenkins/workspace/Puzzle Cypress/cypress/reports/html/.jsons"
Enhance report
Create HTML report`

and it stays that way indefinitely.

All is working well when run on a local machine.

**Comment by LironEr** [comment]
Sorry, but I have no idea what can get wrong there.

You can add to `reporterOptions` `debug: true`, it should create a log file, if you can share the log file it might help to understand why it's stuck

**Comment by AlephCK** [comment]
Hi, 
I had this issue recently while running Cypress Tests + the reporter on GitLab CI. 

I've noticed that having the `autoOpen` flag as `true` might cause the job to get stuck. I had to change it to `autoOpen: false` and after that, the Gitlab CI Job was able to finish successfully with the report created.

Also, before figuring this out, I've tried using `debug: true` while having set `autoOpen: true` and it didn't generated any files while it was stuck.
